core/storage/skill/gae_models.py

```diff
@@ -138,13 +138,7 @@ def _trusted_commit(
             committer_user_settings_model.username
             if committer_user_settings_model else '')
 
-        skill_rights = SkillRightsModel.get_by_id(self.id)
-
-        status = ''
-        if skill_rights.skill_is_private:
-            status = constants.ACTIVITY_STATUS_PRIVATE
-        else:
-            status = constants.ACTIVITY_STATUS_PUBLIC
+        status = constants.ACTIVITY_STATUS_PUBLIC
```

Comment:
Just to check, is this the only possible status?

Do we validate this in prod-validation jobs?

Reply:
Yeah, this is, though since there are no external inputs that can change this (the post_commit_status is set in this function), is there a need for a validation in prod_validation?
Since questions are also similar, and they don't have specific 'public only' validations.

Reply:
Yes, because such methods may be added in the future. In general, the validation checks in prod_validation should check everything that is possible to be checked about the models. So, it would be good to add it.

Reply:
Done